typings/html-renderer.d.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import _CategoryRenderer = require('../lighthouse-core/report/html/renderer/category-renderer.js');
+import _CriticalRequestChainRenderer = require('../lighthouse-core/report/html/renderer/crc-details-renderer.js');
+import _DetailsRenderer = require('../lighthouse-core/report/html/renderer/details-renderer.js');
+import _DOM = require('../lighthouse-core/report/html/renderer/dom.js');
+import _PerformanceCategoryRenderer = require('../lighthouse-core/report/html/renderer/performance-category-renderer.js');
+import _ReportRenderer = require('../lighthouse-core/report/html/renderer/report-renderer.js');
+import _ReportUIFeatures = require('../lighthouse-core/report/html/renderer/report-ui-features.js');
+import _Util = require('../lighthouse-core/report/html/renderer/util.js');
+import _FileNamer = require('../lighthouse-core/lib/file-namer.js');
+
+declare global {
+  var CategoryRenderer: typeof _CategoryRenderer;
+  var CriticalRequestChainRenderer: typeof _CriticalRequestChainRenderer;
+  var DetailsRenderer: typeof _DetailsRenderer;
+  var DOM: typeof _DOM;
+  var getFilenamePrefix: typeof _FileNamer.getFilenamePrefix;
+  var PerformanceCategoryRenderer: typeof _PerformanceCategoryRenderer;
+  var ReportRenderer: typeof _ReportRenderer;
+  var ReportUIFeatures: typeof _ReportUIFeatures;
+  var Util: typeof _Util;
```

Comment:
these add these to the global context for all of our code, even though we only depend on them being global inside the renderer.

An alternative would be to make a second `tsconfig.json` file just for the renderer and compile the main project and the renderer separately. Not sure if it matters right now (but it might be more of a pain to have two separate `tsc` runs, don't know)